Tools/pyproject.toml

```
@@ -0,0 +1,3 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = ["setuptools", "setuptools_scm"]
```

```suggestion
requires = ["setuptools>=42", "setuptools_scm[toml]>=3.4"]
```
Copied from here:
https://github.com/opendatacube/datacube-core/blob/develop/pyproject.toml

In particular `[toml]` part is quite important

---

Thanks!